.github/workflows/bats-tests.yml

```
@@ -14,7 +14,7 @@ jobs:
 
       - uses: shivammathur/setup-php@v2
         with:
-          php-version: ${{ vars.PHP_VERSION }}
+          php-version: '${{ vars.PHP_VERSION }}
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 17: found unexpected end of stream

(syntax-check)